net/nebula/Makefile

```diff
@@ -69,53 +70,49 @@ define Package/nebula-service
   PKGARCH:=all
 endef
```

Comment:
I found the same variable was defined multiple times, this can be avoided by creating a shared recipe like:
```
define Package/nebula/Default
  SECTION:=net
  CATEGORY:=Network
  URL:=https://docs.openwrt.melmac.net/nebula/
end
```

then call it:
```
define Package/nebula
  $(call Package/nebula/Default)
  TITLE:=nebula
  ...
endef
```

the same applies to descriptions:
```
define Package/nebula/description/Default
  Nebula is a scalable overlay networking tool with a focus on performance, simplicity
  and security. It lets you seamlessly connect computers anywhere in the world.
endef
```

```
define Package/nebula/description
  $(call Package/nebula/description/Default)
  This package contains only nebula binary. Unless you want to start nebula manually,
  you may want to also install *either* 'nebula-service' *or* 'nebula-proto' package.
endef
```